src/strings/tests/main_test.bal

```diff
@@ -0,0 +1,104 @@
+// Copyright (c) 2019 WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+//
+// WSO2 Inc. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+import ballerina/test;
+import ballerina/io;
+
+# Before Suite Function
+
+string [] ipList = ["172.300.9.28", "172.8.9.28"];
+
+# Test function
+
+@test:Config{}
+function testEmailValidation () {
+    io:println("-----------------Test case for isEmail method------------------");
```

Comment:
Shall we change io:println() to log:info() ?

Reply:
Since this is test, we don't have to keep the history in the log file.